src/frontend/efiling-demo/src/components/base/button/Button.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import PropTypes from "prop-types";
+import "./Button.css";
+
+export const Button = ({ onClick, label }) => (
+  <div className="outline">
```

Comment:
Non-blocking question: is it weird that this file isn't formatted like your other html? like the storyshots.test.js.snap above?

Reply:
not too sure I understand the question unfortunately